src/App.js

```diff
@@ -1,25 +1,21 @@
-import logo from './logo.svg';
+import React from 'react';
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import Books from './components/Books';
+import Catagories from './components/Categories';
```

Comment:
- [ ] Please change the name of this class to its original which is `Categories`. The page does not render anything because `Catagories` does not exist.